src/framework/annotator_mgr.cpp

```diff
@@ -86,12 +86,13 @@ namespace uima {
     /*       Implementation                                                    */
     /* ----------------------------------------------------------------------- */
 
-    AnnotatorManager::AnnotatorManager(internal::AggregateEngine & rEngine) :
-        iv_pEngine(& rEngine),
-        iv_vecEntries(),
-        iv_bIsInitialized(false),
-        iv_uiNbrOfDocsProcessed(0)
-        /* ----------------------------------------------------------------------- */{
+    AnnotatorManager::AnnotatorManager(internal::AggregateEngine & rEngine) : iv_pEngine(&rEngine),
```

Comment:
changing the white space here creates a diff that's difficult to appreciate. When working with legacy code don't do that, it is important to understand when the change got added rather than just making it look prettier.